python/cudf/dataframe/series.py

```diff
@@ -1113,6 +1113,43 @@ def product(self, axis=None, skipna=True, dtype=None):
         assert axis in (None, 0) and skipna is True
         return self._column.product(dtype=dtype)
 
+    def cummin(self, axis=0, skipna=True):
+        """Compute the cumulative minimum of the series"""
+        assert axis in (None, 0) and skipna is True
+        return Series(self._column._apply_scan_op('min'), name=self.name,
+                      index=self.index)
```

Comment:
Should this be typecast to `int64` if integer as well?

Reply:
We can if desirable. Pandas only typecasts the cumprod/cumsum operations to int64, which I would think is to avoid any potential OverflowError.